src/Microsoft.AspNet.Mvc.TagHelpers/Internal/FileVersionProvider.cs

```diff
@@ -47,7 +47,15 @@ public class FileVersionProvider
         /// </remarks>
         public string AddFileVersionToPath([NotNull] string path)
         {
-            var resolvedPath = path;
+            var resolvedPath = path.Split('?')[0];
```

Comment:
Use `IndexOf` \ `SubString` instead - no point allocating an array if you don't care about the result.